website/docs/i18n/i18n-crowdin.mdx

```diff
@@ -163,7 +163,7 @@ We advise to:
 
 The `api_token_env` attribute defines the **env variable name** read by the Crowdin CLI.
 
-You can obtain a `Personal Access Token` on [your personal profile page](https://crowdin.com/settings#api-key).
+You can obtain a `Personal Access Token` on your personal profile page by clicking your profile photo and selecting "Account Settings - Access Tokens".
```

Comment:
Weird, this is what I see if I follow the link. Everything that was there before.

![image](https://user-images.githubusercontent.com/4408379/133877880-f85b6898-994e-4b28-b6b0-8eeeb7611f18.png)

Reply:
Ah, do you have multiple organizations? I have multiple organizations and the primary organization url is <org_name>.crowdin.com. Visiting https://crowdin.com/settings always bring me to the login page.

Reply:
I think it's cause by multiple organizations. 

BTW, according to https://support.crowdin.com/enterprise/cli/#configuration, i must specify `base_url` in crowdin.yaml to make sure all requests send to the correct API endpoints.

Reply:
No, I just have personal account.

Reply:
Well. that would change a lot of things if the document need to covers many situations includes the multiple accounts. Perhaps we can close the issue and maybe sometime we will add more content in the future. How do you think?

Reply:
Alright, no problem.